content/articles/2019-03-tibble-2-1-0.Rmarkdown

```diff
@@ -0,0 +1,66 @@
+---
+title: tibble 2.1.0
+slug: tibble-2.1.0
+description: >
+    Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! This article describes and motivates the latest minor release.
+date: 2019-03-13
+author: Kirill Müller
+photo:
+  url: https://unsplash.com/photos/yaiy4mCbzw0
+  author: Ganapathy Kumar
+categories: [package]
+---
+
+
+```{r setup, include = FALSE}
+library(tidyverse)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>", error = TRUE)
+options(rlang_trace_top_env = .GlobalEnv)
+
+options(crayon.enabled = TRUE)
+```
+
+```{r setup-hooks, comment="", results="asis", echo=FALSE}
+old_hooks <- fansi::set_knit_hooks(
+  knitr::knit_hooks,
+  which = c("output", "error"),
+  proc.fun = function(x, class) {
+    x <- unlist(strsplit(x, "\n", fixed = TRUE))
+    fansi::html_code_block(fansi::sgr_to_html(fansi::html_esc(x)), class = class)
+  }
+)
+```
+
+
+Version 2.1.0 of the *tibble* package is on CRAN now. Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! Grab the latest version with:
+
+```r
+install.packages("tibble")
+```
+
+This release mostly focuses on the name repair introduced in *tibble* 2.0.1.
+Too late we realized that the implementation doesn't provide satisfactory results in an important setting: unique names for a tibble with empty column names.
```

Comment:
Possible rephrasing (feel free to remix):

We have specific regrets about one aspect of name repair and we think the pros of fixing it outweigh the cons. When a column name is completely absent, the numbered suffix we add becomes the entire name. Originally, we chose `..j` (two dots and a number), but that produces names that require special handling. Since name repair is often something that happens automatically, we think it's best to suffix with `...j` (three dots and a number) and leave people with names that are easier to work with.